### DISCUSSION TITLE
Display all available shares

### DISCUSSION THREAD

**Discussion by stiix13** [epoupon/fileshelter#59] [General]
Hello,
I'd love to be able to display all avaible shares like the "fileshelter-list" command do but directly in the navigator, within a password protected page , could that be a thing ? Should I work on that and post a pull request ?

Thanks

**Comment by epoupon** [comment]
Hello!
Yes why not, would be handy indeed :)

**Comment by stiix13** [comment]
I've created [pull request #63](https://github.com/epoupon/fileshelter/pull/63) with a basic view of current shares if you'd like to add it.
